packages/react-compose/src/utils.ts

```diff
@@ -29,6 +35,16 @@ export const defaultComposeOptions: ComposePreparedOptions = {
   resolveSlotProps: () => ({}),
 };
 
+const mergeClasses = (classes1: ClassDictionary | undefined, classes2: ClassDictionary | undefined) => {
```

Comment:
can we use `function` signature so return type is required

Reply:
see other pr.

Reply:
https://github.com/microsoft/fluentui/pull/13101